config/samples/metal3.io_v1alpha1_hardwareclassification.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: hardwareclassification-sample
   namespace: metal3
   labels:
-    hardwareclassification-sample: matches
+      hardwareclassification-sample: matches
```

Comment:
Ideally it should have 2 spaces, above one is correct

Reply:
sure